easistrain/id15_NXstress/readme.md

```diff
@@ -0,0 +1,53 @@
+# id15_NXstress
```

Comment:
Do you mean simply moving the file or incorporating it into the current docs?

Reply:
Yes and add it to index.rst


```

Documentation
-------------

.. toctree::
    :maxdepth: 2

    thenewfilename
    theory
    reference_frames
    notes
    api
```

This will become visible in https://easistrain.readthedocs.io/en/latest/.

To support markdown you'll need to add this to doc/conf.py

```python
extensions = [
    ....
    "recommonmark",
]

source_suffix = [".rst", ".md"]
```

and this to setup.cfg

```
[options.extras_require]
doc =
    ...
    recommonmark
```

To test documentation building and rendering on your computer:

```
pip install -e .[doc]
sphinx-build ./doc ./html
firefox ./html/index.html
```